File: client-app/src/features/AppConfig/AppConfig.ts
export interface IAppConfig {
	Id: number
	Title: string
	Order:number
	// Type: number
	ConfigTypeId : number;
	Det1: string
	Det2: string
	Det3: string
	Det4: string
	Det5: string
	//defaultSort: string
}

export class AppConfig implements IAppConfig {
	Id: number = 0;
	Title: string = '';
	Order:number = 0;
	// Type: number = 0;
	ConfigTypeId: number = 0;
	Det1: string = '';
	Det2: string = '';
	Det3: string = '';
	Det4: string = '';
	Det5: string = '';
	//defaultSort: string= 'desc';
  
  constructor(init?: IAppConfig) {
    Object.assign(this, init);
  }
}

File: Domain/AppConfig.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Domain
{
    public class AppConfig
    {
		[Key]
      	[DatabaseGenerated(DatabaseGeneratedOption.Identity)]
		public int Id { get; set; }
		public string Title { get; set; }
		public int Order { get; set; }				
		public int Type { get; set; }
		public int ConfigTypeId { get; set; }				
		public virtual AppConfigType ConfigType { get; set; } 
		public string Det1 { get; set; }
		public string Det2 { get; set; }
		public string Det3 { get; set; }
		public string Det4 { get; set; }
		public string Det5 { get; set; }
    }
}

















